express/src/lib/sheets/SheetsService.ts

```diff
@@ -0,0 +1,28 @@
+import SheetsInterface from "./interface";
+
+export default class SheetsService implements SheetsInterface {
+    private implementation!: SheetsInterface;
+    private spreadsheetId: string;
+
+    constructor(spreadsheetId: string) {
+        this.spreadsheetId = spreadsheetId;
+    }
+
+    async init() {
+        if (process.env.USE_STUB_SHEETS === "true") {
```

Comment:
Please use CloudFormation Condition to flip the flag automatically dependent on environment